testsuite/integration/basic/src/test/java/org/jboss/as/test/integration/web/servlet/preservepath/PreservePathFilter.java

```diff
@@ -44,7 +44,7 @@ public void doFilter(ServletRequest servletRequest,
       file.createNewFile();
       try(FileWriter fw = new FileWriter(file)) {
          fw.write("servletPath: " + request.getServletPath());
+         fw.write("\nrequestUrl: " + request.getRequestURL().toString());
```

Comment:
I noticed that this change affects also `getRequestURI` method. It is mentioned also in the analysis. We should probably extend test coverage also for this method too.